### PR TITLE
Add 'open' and 'export' command to CLI

### PR DESCRIPTION
**Pull request by FelipeDefensor**
I know I said we should only fix bugs and improve user experience for v0.5, but I really needed these two commands for some work on my PhD.

### CODE REVIEW

File: tests/test_app.py [hunk 1]
```diff
@@ -288,4 +289,194 @@ def test_slider_timeline_is_created_when_loaded_file_does_not_have_one(self, tls
...
+        file_data = tests.utils.get_blank_file_data()
+        file_data["timelines"] = tl_data
+        file_data["media_metadata"]["media length"] = 100
+
+        tmp_file = tmp_path / "test.tla"
+        tmp_file.write_text(json.dumps(file_data, indent=2))
+        with Serve(Get.FROM_USER_TILIA_FILE_PATH, (True, tmp_file)):
+            actions.trigger(TiliaAction.FILE_OPEN)
+
+        assert settings.get_recent_files()[0] == tmp_file
```
Comment (azfoo) [inline tests/test_app.py:327]: needs to be `Path(settings.get_recent_files()[0])` to not fail on Windows

File: tests/test_app.py [hunk 2]
```diff
@@ -288,4 +289,194 @@ def test_slider_timeline_is_created_when_loaded_file_does_not_have_one(self, tls
...
+        assert len(tls) == 2  # Slider timeline is also created by default
+        assert len(tls[0]) == 3
+
+    def test_open_with_path(self, tilia, tls, tmp_path):
+        tmp_file = tests.utils.get_tmp_file_with_dummy_timeline(tmp_path)
+        post(Post.FILE_OPEN, tmp_file)
+
+        assert settings.get_recent_files()[0] == tmp_file
```
Comment (azfoo) [inline tests/test_app.py:335]: same here

**Review by azfoo — CHANGES_REQUESTED** [review]
seems to work fine. perhaps a check on the file extension on export to prevent silly mistakes?

File: tilia/timelines/base/component.py
[the remark this answers is not in the document]
Comment (FelipeDefensor) [reply tilia/timelines/base/component.py:1]: That would also be handy to encapsulate things like doing maths with measures, which is not always trivial when dealing with changing time signatures. Will do!

[the remark this answers is not in the document]
Comment (FelipeDefensor) [reply tilia/timelines/base/component.py:1]: > Nice abstraction! Think they could probably have more intuitive names. metric position, measure and beat could probably become some kind of data class, especially in the extended one, to include the lengths in it.

I agree the naming could be better. Any ideas?

**Comment by FelipeDefensor** [comment]
So, I got carelles and forogot to initially push a large chunk of the work and then pushed all those WIP commits before squashing, sorry for that. You did review it only after I pushed the missing commits, right?

**Comment by azfoo** [comment]
yes, the review was after the wips

File: tilia/timelines/base/component.py
[the remark this answers is not in the document]
Comment (azfoo) [reply tilia/timelines/base/component.py:1]: maybe point/vertex and segment?

[the remark this answers is not in the document]
Comment (azfoo) [reply tilia/timelines/base/component.py:1]: having -like in the name seems a little redundant. what do you think about dropping it?

[the remark this answers is not in the document]
Comment (FelipeDefensor) [reply tilia/timelines/base/component.py:1]: In any case, I like "segment" as part of the name.

**Review by azfoo — APPROVED** [review]

**Pull request merged**